src/components/Ranking.tsx

```
@@ -0,0 +1,80 @@
+  padding-left: 36px;
+  border-radius: 8px;
+  overflow: hidden;
+`;
+
+const Number = styled.div`
+  width: 54px;
+  height: 54px;
+  border-radius: 27px;
+  background-color: ${Color.blue100};
+  color: ${Color.blue500};
+  ${Font.medium24}
```

Biome (1.9.4)

[error] Do not shadow the global "Number" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)